app/models/application_form.rb

```diff
@@ -289,8 +290,12 @@ def available_application_choices
     continuous_applications? ? application_choices.size - count_unsuccessful_choices : application_choices.size
   end
 
-  def count_unsuccessful_choices
-    application_choices.count { |choice| ApplicationStateChange::UNSUCCESSFUL_STATES.include?(choice.status.to_sym) }
+  def count_unsuccessful_choices(ignore_inactive: false)
+    application_choices.count { |choice| (!ignore_inactive || choice.status.to_sym != :inactive) && ApplicationStateChange::UNSUCCESSFUL_STATES.include?(choice.status.to_sym) }
```

Comment:
There is a lot of negation here. Wouldn't this be more readable:
```ruby
def count_unsuccessful_choices(count_inactive: true)
  application_choices.count { |choice| (count_inactive || choice.status.to_sym != :inactive) && ...
```

Reply:
yes, yes it would